shopping-cart.html

```diff
@@ -215,251 +215,251 @@ <h2 class="can-interes__title">Te podría interesar </h2>
-                <a href="#" class="can-interest__card-link"></a>
+              </li>
+              <li class="can-interest__card">
+                <a href="#" class="can-interest__card-link" aria-label="vizualizar el producto"></a>
                 <div class="can-interest__img">
                   <img src="image/towel.png" alt="Toalla de piso para baño wave">
-                  <span class="can-interest__star can-interest__star--margin-left">&#9733;</span>
-                  <span class="can-interest__rating can-interest__rating--margin-left">4</span>
+                  <span class="can-interest__star can-interest__star--margin-left" aria-hidden="true">&#9733;</span>
+                  <span class="can-interest__rating can-interest__rating--margin-left" aria-label="calificación de productos 4">4</span>
                 </div>
                 <span class="can-interest__text can-interest__text--font-size">STELLLI</span>
                 <span class="can-interest__prod">Toalla de piso para baño wave</span>
-                <p class="can-interest__price can-interest__price--margin-top">$99.99</p>
+                <p class="can-interest__price can-interest__price--margin-top" aria-label="precio de producto 9.99">$99.99</p>
                 <div class="can-interest__row">
-                  <svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
                     <path d="M13.7912 5.42328L6.45784 12.7566L3.12451 9.42328" stroke="#05B999" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                   </svg>
-                  <p class="can-interest__row-text">Envío a domicilio</p>
+                  <p class="can-interest__row-text" aria-label="Este producto esta disponible para ser enviado a domicilio">Envío a domicilio</p>
                 </div>
                 <div class="can-interest__row">
-                  <svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
                     <path d="M13.7912 5.42328L6.45784 12.7566L3.12451 9.42328" stroke="#05B999" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                   </svg>
-                  <p class="can-interest__row-text">Retiro en tienda </p>
+                  <p class="can-interest__row-text" aria-label="Este producto esta disponible para retiro en tienda">Retiro en tienda </p>
                 </div>
                 <div class="can-interest__delivery">
-                  <img class="can-interest__delivery-day" src="image/Entregahoy.png" alt="day"> Entrega hoy
+                  <img class="can-interest__delivery-day" src="image/Entregahoy.png" alt="entrega para hoy"> Entrega hoy
```

Comment:
Siento que esta imagen es decorativa. Tambien le pondria un aria al mensaje asi como le colocaste al de `retiro en tienda`
```suggestion
                  <img class="can-interest__delivery-day" src="image/Entregahoy.png" alt=""> <span aria-label="Este producto esta disponible para entrega hoy">Entrega hoy</span>
```